Source/astcenc_weight_align.cpp

```diff
@@ -135,15 +135,11 @@ static void compute_angular_offsets(
 	int max_angular_steps,
 	float* offsets
 ) {
-	float anglesum_x[ANGULAR_STEPS];
-	float anglesum_y[ANGULAR_STEPS];
-
-	for (int i = 0; i < max_angular_steps; i++)
-	{
-		anglesum_x[i] = 0;
-		anglesum_y[i] = 0;
-	}
-
+	alignas(ASTCENC_VECALIGN) float anglesum_x[ANGULAR_STEPS];
+	alignas(ASTCENC_VECALIGN) float anglesum_y[ANGULAR_STEPS];
+	memset(anglesum_x, 0, max_angular_steps*sizeof(anglesum_x[0]));
```

Comment:
As these are both arrays of known size you can just do `sizeof(anglesum_x)`, etc.

Reply:
that would memset the whole array, whereas only max_angular_steps portion is needed